docker/kubernetes/README.md

```diff
@@ -0,0 +1,234 @@
+  associated_node_id: bubba
+  metadata:
+    gameroom/first-name: bob
+    gameroom/organization: Bubba Bakery
+```
+
+### Step 3: Create ConfigMaps for KeyRegistry and NodeRegistry
+
+`$ kubectl create configmap key-registry --from-file key-registry.yaml`
+
+Download the NodeRegistry file: [node-registry.yaml](https://raw.githubusercontent.com/Cargill/splinter/master/docker/kubernetes/node-registry.yaml).
+
+`$ kubectl create configmap node-registry --from-file node-registry.yaml`
+
+Verify the ConfigMaps were created:
+
+```
+$ kubectl get cm
+NAME            DATA   AGE
+key-registry    1      30s
+node-registry   1      2m43s
+```
+
+You can inspect the values of the ConfigMaps by running
+`kubectl describe cm <configmapname>`. For example:
+
+```
+$ kubectl describe cm node-registry
+Name:         node-registry
+Namespace:    default
+Labels:       <none>
+Annotations:  <none>
+
+Data
+====
+node-registry.yaml:
+----
+---
+- identity: "acme"
+  metadata:
+    organization: "Acme Corporation"
+    endpoint: "tls://acme.default.svc.cluster.local:8044"
+- identity: "bubba"
+  metadata:
+    organization: "Bubba Bakery"
+    endpoint: "tls://bubba.default.svc.cluster.local:8044"
+
+Events:  <none>
+```
+
+### Step 4: Starting Gameroom
+Apply the `arcade.yaml` manifest:
+
+```
+$ kubectl apply -f https://raw.githubusercontent.com/Cargill/splinter/master/docker/kubernetes/arcade.yaml
+
+deployment.apps/acme created
+service/acme-splinterd created
+service/acme-http created
+deployment.apps/bubba created
+service/bubba-splinterd created
+service/bubba-http created
+```
+
+Verify the pods started correctly:
+
+```
+$ kubectl get pods
+NAME                     READY   STATUS      RESTARTS   AGE
+acme-7575f75d6d-trddh    5/5     Running     0          6m10s
+bubba-5d5f554fdb-l9nr4   5/5     Running     0          6m9s
+gameroom-keys-l42fz      0/1     Completed   0          56m
+```
+
+### Step 5: Create Users and Log In to the Web App
+
+Run `minikube service` to open the Acme Web App.
+
+```
+$ minikube service acme-http
+|-----------|-----------|-------------|---------------------------|
+| NAMESPACE |   NAME    | TARGET PORT |            URL            |
+|-----------|-----------|-------------|---------------------------|
+| default   | acme-http | http        | http://192.168.64.3:30160 |
+|-----------|-----------|-------------|---------------------------|
+Opening service default/acme-http in default browser...
+```
+![alt text](images/acme-1-launch.png "Acme gameroom homepage")
+
+In the upper left, click "Register".
+
+Fill out the registration form with values you'll remember. For the
+`Private Key` field, enter the private key information generated for Alice in
+Step 1.
+
+![alt text](images/acme-2-register.png "Acme registration page")
+
+You're logged in as Alice now.
+![alt text](images/acme-3-loggedin.png "Alice logged in page")
+
+Repeast steps for Bubba/Bob.
```

Comment:
Typo (Repeat), and the other company is "Bubba Bakery" (avoid using just "Bubba").

Also, this sentence feels a bit too terse.  My suggestion:  "Repeat these steps to launch the Bubba Bakery web app and register with Bob's private key."